packages/app/src/Components/Embed/TidalEmbed.tsx

```diff
@@ -55,7 +55,7 @@ const TidalEmbed = ({ link }: { link: string }) => {
   }
   const iframe = (
     // eslint-disable-next-line react/no-unknown-property
-    <iframe src={source} style={extraStyles} width="100%" title="TIDAL Embed" frameBorder={0} credentialless="" />
+    <iframe src={source} style={extraStyles} width="100%" allow="encrypted-media *; clipboard-write *; clipboard-read *" sandbox="allow-scripts allow-popups allow-forms allow-same-origin"  title="TIDAL Embed" frameBorder={0} credentialless="" />
```

Comment:
`allow-popups` = for sharing to facebook / x / messanger
`allow-forms` = to be able to close the share dialog (HTMLDialogElement with a HTMLFormElement)
`allow-same-origin` = needed for CORS
`allow-scripts` = needed for JS